iron-form.html

```diff
@@ -280,8 +280,16 @@
         if (!this._useValue(el) ||
             (el.hasAttribute('is') && json[el.name])) {
           continue;
+        } else if (el.tagName.toLowerCase() === 'select' && el.multiple) {
```

Comment:
`type="file"` doesn't really work at all (you have to serialize it yourself, since there's no `value` per se)